src/components/EditDisplayName/EditDisplayName.tsx

```diff
@@ -0,0 +1,113 @@
+  const { user } = useAuthContext()
+  const [currentDisplayName, setDisplayName] = useState<string>('')
+  const [submissionStatus, setSubmissionStatus] = useState<boolean>(false)
+
+  const greeting = userDisplayName ? `Welcome, ${userDisplayName}` : 'Welcome!'
+
+  const updateDisplayName = () => {
+    if (!user) {
+      return
+    }
+    handleEditDisplayName(user.userId, currentDisplayName)
+    setSubmissionStatus(true)
+  }
+
+  const resetDisplayName = () => {
+    setDisplayName('')
+    setSubmissionStatus(false)
+  }
+
+  return (
+    <Grid row>
+      <Grid col="fill">
+        <h3>Update name and rank:</h3>
+
+        <div className={styles.updateNameAndRank}>
+          <label htmlFor="something">Current welcome display title:</label>
+          <h2>{greeting}</h2>
+
+          <label htmlFor="displayName">
+            <strong>*</strong>Update name
+          </label>
+
+          <Grid row gap={4}>
+            <Grid tablet={{ col: 4 }}>
+              <input
+                className="usa-input"
+                type="text"
+                name="displayName"
+                placeholder="Enter a display name"
+                value={currentDisplayName}
+                maxLength={1000}
+                onChange={(event) => setDisplayName(event.target.value)}
+                data-testid="nameInput"
+              />
+            </Grid>
+            <Grid row tablet={{ col: true }} gap={5}>
+              <button
+                type="button"
+                className="usa-button usa-button--unstyled"
+                onClick={resetDisplayName}>
+                Cancel
+              </button>
+              <button
+                type="button"
+                className="usa-button"
+                disabled={currentDisplayName === '' ? true : false}
+                onClick={updateDisplayName}>
+                Save
+              </button>
+            </Grid>
+          </Grid>
+
+          {submissionStatus && (
+            <Grid row gap={4}>
+              <Grid tablet={{ col: 4 }}>
+                <Alert
+                  className={styles.successMessage}
+                  type="success"
+                  headingLevel={4}>
+                  New title has been saved
+                </Alert>
+              </Grid>
+            </Grid>
+          )}
+        </div>
+      </Grid>
+
+      {/* <Grid col="auto">
+        <div className={styles.updateOfficialRecords}>
+          <h3>
+            Do your records, including your CAC, currently display and incorrect
+            name or rank?
+          </h3>
+          <p>
+            We can help with that. In order to change or update the name on your
+            official records, you’ll need to call the office of IDs and Naming
+            Stuff, or IDANS.
+          </p>
+          <p>
+            You can reach them by phone, Monday through Saturday, 9am - 5pm GMT
+            at <insert phone number>
+          </p>
+        </div>
+      </Grid> */}
```

Comment:
Correct. Although in the design there is a button to submit an official form on behalf of the user, but since we don't have that flow yet, I believe we briefly discussed just placing the phone number. Would need more input from design on this.

Reply:
Cool, I'm fine leaving this comment here as a TODO.